src/Sql/dbo/Stored Procedures/Provider_Create.sql

```diff
@@ -9,6 +9,9 @@
     @BusinessTaxNumber NVARCHAR(30),
     @BillingEmail NVARCHAR(256),
     @BillingPhone NVARCHAR(50) = NULL,
+    @Gateway TINYINT,
+    @GatewayCustomerId VARCHAR(50),
+    @GatewaySubscriptionId VARCHAR(50),
```

Comment:
❌ These **will** break functionality if the proc goes before calling code. They'll need to be at the end and have defaults. Please repeat this fix everywhere relevant.

Reply:
Yes please. We want uniformity.

Reply:
Addressed here: https://github.com/bitwarden/server/pull/3875/commits/bc0a567530148ed9432d72cc92e8bec6be385bb4